bin/sam-coverage.py

```diff
@@ -58,7 +58,9 @@ def filterRead(read):
     offsetsCovered = len(coveredOffsets[referenceId])
     referenceLength = referenceLengths[referenceId]
     print('%s: length %d, covering reads %d, covered sites %d (%.4f%%), '
-          'coverage depth %.4f' %
+          'mean coverage depth %.4f (min: %d, max: %d)' %
           (referenceId, referenceLength, len(coveringReads[referenceId]),
            offsetsCovered, offsetsCovered / referenceLength * 100.0,
-           sum(coveredOffsets[referenceId].values()) / referenceLength))
+           sum(coveredOffsets[referenceId].values()) / referenceLength,
+           min(coveredOffsets[referenceId].values()),
+           max(coveredOffsets[referenceId].values())))
```

Comment:
Can `coveredOffsets[referenceId]` be empty? If so, these lines will both raise a `ValueError` in that case. The `sum` line above doesn't because `sum` returns `0` on an empty list. I think you could just pass `default=0` in both cases?